README.md

```diff
@@ -56,58 +56,73 @@ We currently do not:
 
 [Docker](https://www.docker.com/) is the prefered development environment.  Ensure you have Docker Engine installed or otherwise can run containers.
 
-First, open `ci/.docker-env.example`, fill in values as desired, and save as `ci/.docker-env`.  Then build the "notification_api" Docker image by running this command:
+`.local.env` contains all necessary environmental variables and is read automatically in local development and github actions that do not deploy to AWS.
 
-```bash
-docker-compose -f ci/docker-compose-local.yml build app
-```
 
-**Rebuild notification_api whenever Dockerfile.local or requirements-app.txt changes.**
+**Rebuild `notification_api` whenever Dockerfile or poetry.lock changes.**
 
-The associated container will have your local notification-api/ directory mounted in read-only mode, and Flask will run in development mode.  Changes you make to the code should trigger Flask to restart on the container.
+The associated container will have your local notification-api/ directory mounted in read-write mode, and Flask will run in development mode.  Changes you make to the code should trigger Flask to restart on the container. The volume specified in `ci/docker-compose-local.yml` allows the container to read and write from your local file system. If you wish to isolate the built container from your filesystem simply comment out the volume, but you will have to also comment out the `ENTRYPOINT` because `scripts/save_certificate.sh` saves data to the filesystem. This can be useful if working on the Dockercontainer, to verify the expected data is in the expected spot when deployed to a non-local environment.
```

Comment:
```suggestion
The associated container will have your local notification-api/ directory mounted in read-write mode, and Flask will run in development mode.  Changes you make to the code should trigger Flask to restart on the container. The volume specified in `ci/docker-compose-local.yml` allows the container to read and write from your local file system. If you wish to isolate the built container from your filesystem, comment out the volume, but you will have to also comment out the `ENTRYPOINT` because `scripts/save_certificate.sh` saves data to the filesystem.  When working on the Docker container, this can be useful to verify the expected data is in the expected spot when deployed to a non-local environment.
```

Reply:
Will address in #1692